storybookjs/storybook
# Stop add-on Draggable from overlapping the vertical scrollbar when stories overflow

Issue: #17202 

fix the draggable overlaying the vertical scrollbar when stories overflow at the bottom, making the story un-scrollable.

## What I did
There was a negative margin on the Draggable component that is responsible for resizing the add-on panel when in side-panel mode.
This margin pushed the Draggable to the left by 10px, which caused it to overlap with the vertical scrollbar for scrolling down a long story.

I have removed this margin, which puts the Draggable in the correct spot.

Below is an image illustrating the old- (yellow) & new (blue) positions.

![Group](https://user-images.githubusercontent.com/12213529/157265224-a11fd498-0216-4be2-ae76-73b8c0305296.png)


https://user-images.githubusercontent.com/12213529/158625721-d1e4d46a-bbfc-4cca-a095-0dff9c36de6d.mov



## How to test
1. Create a story that overflows on the bottom.
2. View the story.
3. Try to scroll down with the right-hand scrollbar. (should be able to use the entire width of the scrollbar)
4. Try to resize the add-on panel in side-panel mode. (should be at the correct position, not overlapping anything)


If your answer is yes to any of these, please make sure to include it in your PR.
